shell/probe/raindrop.ps1

```diff
@@ -33,13 +33,24 @@ while ($true) {
             "dat" = $dat
             "version" = "2.1"
         } -UseBasicParsing -MaximumRedirection 0 -ErrorAction SilentlyContinue
-        
+
         $uuid = $task.content -replace ".*?([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}).*", '$1'
         $auth = $task.content -replace '^[^/]*//([^/]*)/.*', '$1'
 
         if ($uuid -and $auth -eq $ca) {
             Invoke-WebRequest -Uri $task.content -OutFile (New-Item -path "$dir\$uuid.exe" -Force ) -UseBasicParsing
-            $code = Execute "$dir\$uuid.exe"
+            $stdoutTempFile = New-TemporaryFile
+            $stderrTempFile = New-TemporaryFile
+            $code = Execute "$dir\$uuid.exe" $stdoutTempFile $stderrTempFile
+            $stdout = Get-Content -Path $stdoutTempFile
```

Comment:
I could `Write-Host` and that would write directly to stdout instead of returning, but I wasn't sure if that had any implications for the calling service and I don't have a great way to test that before shipping so I went with what I knew would work.

Reply:
doing some testing on this to determine what is possible.

Reply:
`Write-Host` works fine so I moved these into exec